support-frontend/assets/pages/paper-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -258,12 +255,6 @@ function CheckoutForm(props: PropTypes) {
             >
               Continue to payment
             </Button>
-            <DirectDebitPopUpForm
```

Comment:
is this being used currently?

Reply:
No it was a duplicate of this:
https://github.com/guardian/support-frontend/blob/909e45760fee9c6a30104763e24b88330d95d752/support-frontend/assets/components/subscriptionCheckouts/paymentMethodSelector.jsx#L84